indra/assemblers/english/assembler.py

```diff
@@ -75,6 +76,8 @@ def make_model(self):
                 stmt_strs.append(_assemble_gap(stmt))
             elif isinstance(stmt, ist.Conversion):
                 stmt_strs.append(_assemble_conversion(stmt))
+            elif isinstance(stmt, PybelEdge):
+                stmt_strs.append(_assemble_pybel_edge(stmt))
```

Comment:
I am not convinced that handling PybelEdges here is the best option. Would it be possible to iterate over edges in a path in the reporting module, and deciding if the EnglishAssembler or a separate PyBEL-edge to English converter should be called for the given edge?

Reply:
That makes sense. I will create a separate function converting PybelEdge to English in reporting module. For now the only place it's going to be called is in EMMAA, so it can be easily handled there.